.github/workflows/haskell.yml

```
@@ -103,6 +106,13 @@ jobs:
             cabal_version: 3.6.2.0
             cabal_project: cabal.project.streamly
             ignore_error: true
+          - name: 9.0.2-streamly-core-sdist
+            ghc_version: 9.0.2
+            runner: ubuntu-latest
+            build: cabal
+            cabal_version: 3.6.2.0
+            ignore_error: true
```

We should not ignore error on this build.